VMEncryption/main/BekUtilFileImpl.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+#
+# Azure Disk Encryption For Linux extension
+#
+# Copyright 2016 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import traceback
+from AbstractBekUtilImpl import AbstractBekUtilImpl
+import sys
+import os
+
+class BekUtilFileImpl(AbstractBekUtilImpl):
+    """
+    Utility class to store passphrase in a keyfilepath.
+    """
+    def __init__(self,diskutil,logger):
+        self.keyfilePath = "/var/lib/azure_disk_encryption_config/"
+        self.logger = logger
+        self.disk_util=diskutil
+        self.keyPathNotFound = "Keyfile path is not valid, path: {0}".format(self.keyfilePath)
+    
+    def store_bek_passphrase(self, encryption_config, passphrase):
+        bek_filename = encryption_config.get_bek_filename()
+        try:
+            self.disk_util.make_sure_path_exists(self.keyfilePath)
+            self.store_passphrase(key_File_Path=self.keyfilePath,
+                                  bek_filename=bek_filename,
+                                  passphrase=passphrase)            
+        except Exception as e:
+            message = "Failed to store BEK in BEK VOLUME with error: {0}".format(traceback.format_exc())
```

Comment:
This message needs to be updated. I believe the passphrase is not stored in BEK volume here.